lib/storage.js

```diff
@@ -46,6 +46,17 @@ Store.prototype._createAppRef = function (id, fbRef, data) {
 
     function newBlock(blockId) {
         var block = blocks[blockId];
+        var localBlock = block;
```

Comment:
you should do 
```
if (!block) return;
var clone = clone(block);
```

or else you will be directly modifying the original `blocks` object